app/views/privacy/privacy.njk

```diff
@@ -5,322 +5,371 @@
 {% endblock %}
 
 {% block mainContent %}
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-l">Privacy notice</h1>
+  <div class="govuk-grid-column-one-third sub-navigation-container">
+    <nav class="sub-navigation" data-click-events data-click-category="Content" data-click-action="Link clicked"
+      data-cy="sub-navigation">
+      <ol itemscope itemtype="http://schema.org/ItemList">
+        <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+          <a class="govuk-link" itemprop="item" href="#who-we-are">
+            <span itemprop="name">Who we are</span></a>
+        </a>
```

Comment:
Can this file be re-formated? Closing tag for the anchor (`</a>`) doesn't align with the opening one. Similar formatting is to be done through out.

Reply:
Both VSCode and Intelli J could not format that bit properly.
I copied this from the Product page repo - so have now manually tweaked it.

However, Intelli J formatting fixed some other indentation issues.  So used it to format the entire file.